Source/WebKit/UIProcess/WebAuthentication/Cocoa/LocalAuthenticator.mm

```diff
@@ -60,6 +60,14 @@
 
 #if USE(APPLE_INTERNAL_SDK)
 #import <WebKitAdditions/LocalAuthenticatorAdditions.h>
+
+static BOOL shouldUseAlternateKeychainAttribute()
+{
+    if (![WebKit::getASCWebKitSPISupportClass() respondsToSelector:@selector(shouldUseAlternateKeychainAttribute)])
+        return NO;
+
+    return [WebKit::getASCWebKitSPISupportClass() shouldUseAlternateKeychainAttribute];
```

Comment:
Why are you duplicating all of this code from `Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm`? You should have one copy of this code in a place you can use from both of these compilation units.

Reply:
I put it in LocalAuthenticator.h within the WebKit namespace.